cmd/Makefile

```diff
@@ -1,7 +1,7 @@
 BINARIES = gen_release_notes gen_release_report backport semv rancher_release test_coverage upstream_go_version rke2_release release
 ARCHS ?= amd64 arm64
 OSs ?= linux darwin freebsd
-GO_COMPILE = GOOS=$${os} GOARCH=$${arch} CGO_ENABLED=1 $(GO) build -tags $(TAGS) -v -ldflags "$(LDFLAGS)" -o $@-$${os}-$${arch}
+GO_COMPILE = GOOS=$${os} GOARCH=$${arch} $(GO) build -tags $(TAGS) -v -ldflags "$(LDFLAGS)" -o $@-$${os}-$${arch}
```

Comment:
I thought we needed this on since we're using the hardened image?

Reply:
Building in the hardened image would also work, but right now builds are being done of the Ubuntu runner vm, not in a container.

Reply:
I can see I'm wrong about that. I'll cancel review for now.

Reply:
I added FreeBSD since I occassionally work from my FreeBSD machine and provides a bulk of MacOS' system but we can drop it entirely. It's not even slightly a requirement.